### DISCUSSION TITLE
Adobe Flash CS6 ( Big Sur Icon )

### DISCUSSION THREAD

**Discussion by sulflix** [elrumo/macOS_Big_Sur_icons_replacements#1490] [Icon request]
SVG of the original Flash CS6 icon: https://static.wikia.nocookie.net/logopedia/images/5/5e/Flash_2012.svg/revision/latest?cb=20210525153018

**Comment by bunkerdesign** [comment]
![Adobe Flash](https://github.com/user-attachments/assets/1f6adeb8-043b-4f94-b5a3-d91d1a55b0b4)
![Adobe Flash v2](https://github.com/user-attachments/assets/91da89c9-56e2-4d9f-a2d6-6488b017069f)

here you go

**Comment by shiro6x** [comment]
Borders too thick, don't match with the others like Premiere and AfterEffects

**Reply by bunkerdesign** [reply]
> Borders too thick, don't match with the others like Premiere and AfterEffects

Borders are slightly thinner than on the CS6 one. I don't know which set of icons you are using so you are free to make a matching icon yourself
![Adobe Flash v2](https://github.com/user-attachments/assets/4a2257d7-3e18-44cf-9777-9b2e91d7c811)
